osdk/src/base_crate/x86_64.ld.template

```diff
@@ -65,6 +78,20 @@ SECTIONS
         __ktest_array_end = .;
     }
 
+    . = ALIGN(0x1000);
+    __cpu_local_start = .;
+    .cpu_local              : AT(ADDR(.cpu_local) - KERNEL_VMA) {  
+        KEEP(*(SORT(.cpu_local)))
+    }
+    . = ALIGN(0x1000);
+    __cpu_local_end = .;
+
+    /* Reserve a section with size __cpu_local_end-__cpu_local_start filled with zeroes */
+    __bsp_local_start = .;
+    . += __cpu_local_end - __cpu_local_start;
+    . = ALIGN(0x1000);
+    __bsp_local_end = .;
+
```

Comment:
Maybe `.cpu_local` is more like `.rodata` (read only), and `__bsp_local_start` is akin to `.bss` (rw but no file size). Place them together if not possible to merge them in order to make ELF segments more unified. It's currently a bit scattered now.